### DISCUSSION TITLE
No download files button+ not getting the benefits

### DISCUSSION THREAD

**Discussion by Mohamed-Aziz-Yahia** [community/community#93745] [GitHub Education] [answered]
### Hi everyone,

Hey everyone, I hope you are doing well.
When applying for the student pack I crossed with 2 problems:
First, I didn't find the uploading files button, eventhough I live near to my college. Taking the photo with only front camera option was a headache, and the picture wasn't that good.
Second, I got approved with that bad picture, and now I am wondering if I am going to receive my benefits, especially that I waited for more then 3 days and there's nothing. 
![Screenshot from 2024-01-21 08-02-44](https://github.com/community/community/assets/157042269/758bcb8f-845c-45c7-a22a-64b80f866bfe)
Any solutions?

**Comment by brenoepics** [comment, marked as answer]
Hey, GitHub usually takes a while to make its benefits available, when it is available it will look purple like this:
![image](https://github.com/community/community/assets/59066707/476fefd3-2a1f-46a4-961d-2b9a0e3c3738)
for me and other friends it took about 4-5 days.

**Reply by Mohamed-Aziz-Yahia** [reply]
Mmm I see, when they said 72h I thought if I didn't get it by that time I'll never do .

**Reply by brenoepics** [reply]
If they reject it will show red

**Reply by brenoepics** [reply]
I think it must have already been approved, try taking a look again.

**Reply by Mohamed-Aziz-Yahia** [reply]
How did you know? Yes I just saw the email they approved the application
